examples/example-gateway/endpoints/bar/endpoint-config.json

```diff
@@ -20,7 +20,8 @@
 	},
 	"dependencies": {
 		"client": [
-			"bar"
+			"bar",
+			"quux"
```

Comment:
why this change ?

Reply:
Same. To demostrate creating a custom tchannel client.

Reply:
understood, got it.